modules/apps/transfer/keeper/keeper_test.go

```diff
@@ -30,13 +30,15 @@ type KeeperTestSuite struct {
 	chainA *ibctesting.TestChain
 	chainB *ibctesting.TestChain
 	chainC *ibctesting.TestChain
+	chainD *ibctesting.TestChain
```

Comment:
Some food for thought for a future refactor or cleanup.. Maybe we should consider a `ForwardingTestSuite` for forwarding tests, since spinning up 4 in process apps is wasteful for most other transfer tests.

Reply:
That's a very reasonable take. I'll create an issue for this

Reply:
yeah I think a new test suite for this will be much cleaner